clerk/types/msg.go

```diff
@@ -59,6 +60,11 @@ func (msg MsgEventRecord) ValidateBasic() sdk.Error {
 	if msg.TxHash.Empty() {
 		return sdk.ErrInvalidAddress("missing tx hash")
 	}
+
+	if len(msg.Data) > helper.MaxStateSyncSize {
+		return ErrSizeExceed("length is larger than 100000 bytes")
```

Comment:
Let's use the constant `MaxStateSyncSize` here.

Reply:
Done